src/Console/AbstractDealwebCommand.php

```diff
@@ -112,25 +117,25 @@ protected function parseVerbosity($level = null)
 
     public function startProcess($message)
     {
-        if (! $this->output->isVerbose()) {
-            $this->output->write(str_pad("   - " . $message, $this->columns - 15, ' ', STR_PAD_RIGHT));
+        if (!$this->output->isVerbose()) {
+            $this->output->write(str_pad('   - '.$message, $this->columns - 15, ' ', STR_PAD_RIGHT));
         }
     }
 
     public function endProcess($status = true)
     {
         if ($this->output->isVerbose()) {
-            $this->output->write(" <= ");
+            $this->output->write(' <= ');
         }
 
         if ($status) {
-            $this->output->writeln("[   <info>Ok</info>   ]");
+            $this->output->writeln('[   <info>Ok</info>   ]');
         } else {
-            $this->output->writeln("[ <error>Failed</error> ]");
+            $this->output->writeln('[ <error>Failed</error> ]');
```

Comment:
Call to method `writeln` from undeclared class `\Symfony\Component\Console\Output\OutputInterface`